pkgs/racket-doc/scribblings/foreign/types.scrbl

```diff
@@ -1402,6 +1402,34 @@ expects arguments for both the super fields and the new ones:
 #:changed "6.1.1.8" @elem{Added @racket[#:offset] for fields.}
 #:changed "6.3.0.13" @elem{Added @racket[#:define-unsafe].}]}
 
+@defproc[(compute-offsets [types (listof ctype?)]
+                          [alignment (or/c #f 1 2 4 8 16) #f]
+                          [declare (listof (or/c #f exact-integer?)) '()])
+         (listof exact-integer?)]{
+                                  
+ Given a list of types in a C struct type, return the offset
+ of those types.
+
+ @racket[types] is a list of types of the C struct. The same
```

Comment:
Please don't start a sentence with a lowercase identifier (http://docs.racket-lang.org/style/reference-style.html section 5.3). The second sentence isn't complete.

Reply:
Fixed, thanks.